charts/prometheus/templates/deploy.yaml

```diff
@@ -296,10 +296,29 @@ spec:
       tolerations:
 {{ toYaml .Values.server.tolerations | indent 8 }}
     {{- end }}
-    {{- if .Values.server.affinity }}
+    {{- if or .Values.server.affinity .Values.server.podAntiAffinity }}
       affinity:
-{{ toYaml .Values.server.affinity | indent 8 }}
     {{- end }}
+      {{- with .Values.server.affinity }}
+{{ toYaml . | indent 8 }}
+      {{- end }}
```

Comment:
```suggestion
      {{- with .Values.server.affinity }}
      {{ toYaml . | indent 8 }}
      {{- end }}
```

Reply:
I was following the precedent in the rest of the file. Every other use of toYaml had no indentation before the template expression. Do you still want me to make this suggested change?

Reply:
Fixed in ee083c1d

Reply:
In truth, the style is not entirely consistent.